src/utils.js

```diff
@@ -24,17 +24,17 @@ function toBasicTokenUnit (val, decimals = 18, precision = 3) {
   let base = new BN(10).pow(new BN(decimals - precision))
   let precisionBase = new BN(10).pow(new BN(precision))
   let rv = parseInt(val * precisionBase.toNumber())
-  return new BN(rv).pow(base)
+  return (new BN(rv).pow(base)).toString()
 }
 
 async function getGasCost (txObj) {
   const _web3 = new Web3(new Web3.providers.HttpProvider(infuraApi))
-  let gasPrice = await _web3.eth.getGasPrice()
+  let price = await _web3.eth.getGasPrice()
   let gas = (await _web3.eth.estimateGas(txObj)).toString()
-  let costInWei = (new BN(gasPrice).mul(new BN(gas))).toString()
-  let costInEther = _web3.utils.fromWei(costInWei, 'ether')
+  let costInBasicUnit = (new BN(price).mul(new BN(gas))).toString()
+  let costInStandardUnit = _web3.utils.fromWei(costInBasicUnit, 'ether')
 
-  return { gasPrice, gas, costInWei, costInEther }
+  return { price, gas, costInBasicUnit, costInStandardUnit }
```

Comment:
inconsistent type for costInStandardUnit, here we have **string**,  but we have **float** in other places, see https://github.com/ventureum/Chainsfer/blob/fa1cfab611a818c0f368af9dea63361ea0fed612/src/actions/transferActions.js#L52

Reply:
fixed